server/src/tree-sitter/analyzer.ts

```diff
@@ -69,6 +69,10 @@ export default class Analyzer {
     return this.uriToAnalyzedDocument[uri]?.variableExpansionSymbols ?? []
   }
 
+  public removeLastScanResultForUri (uri: string): void {
+    this.uriToLastScanResult[uri] = undefined
```

Comment:
you should rather do `delete this.uriToLastScanResult[uri]`

Reply:
deleting dynamically computed key is not recommended by eslint